source/css/tailwind.css

```diff
@@ -17,19 +17,19 @@
 }
 
 h5 {
-  @apply text-xl;
+  @apply text-xl; 
 }
 
 h6 {
-  @apply text-lg;
+  @apply text-lg; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/Driminary/thomas-anderson.net/pullRequest?prid=5875174)